api/v1alpha1/vaultdynamicsecret_types.go

```diff
@@ -18,10 +18,25 @@ type VaultDynamicSecretSpec struct {
 	Namespace string `json:"namespace,omitempty"`
 	// Mount path of the secret's engine in Vault.
 	Mount string `json:"mount"`
+	// RequestHTTPMethod to use when syncing Secrets from Vault.
+	// Setting a value here is not typically required.
+	// If left unset the Operator will make requests using the GET method.
+	// In the case where Params are specified the Operator will use the PUT method.
+	// Please consult https://developer.hashicorp.com/vault/docs/secrets if you are
+	// uncertain about what method to use.
+	// +kubebuilder:validation:Enum={GET,PUT}
+	RequestHTTPMethod string `json:"requestHTTPMethod,omitempty"`
+	// uncertain about what 'path' should be set to.
```

Comment:
```suggestion
```

Reply:
Fixed in 53b3b367ef1a78cb9d978916f545fc943cd081e6